DESCRIPTION

```diff
@@ -20,10 +20,12 @@ LazyData: true
 Suggests: 
     covr,
     testthat (>= 2.1.0),
-    httptest
+    httptest,
+    dplyr
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 6.1.1
 Imports: 
     utils,
     httr,
-    jsonlite
+    jsonlite,
+    data.table
```

Comment:
Importing `data.table` just to use 1 function is a no for me -- We are making our package much larger then required. We can use `do.call(rbind, <list>)` instead.